pyseir/inference/model_fitter.py

```diff
@@ -703,12 +703,11 @@ def _persist_results_per_state(state_df):
             pickle.dump(county_series.mle_model, f)
 
 
-def build_county_list(state: str) -> List[str]:
+def build_county_list(state: str, df_whitelist: pd.DataFrame) -> List[str]:
```

Comment:
small nit: in one place this is called `df_whitelist` in others its `whitelist_df`

Reply:
Done